tests/Integration/Momento.Sdk.Tests/CacheEagerConnectionTest.cs

```diff
@@ -21,30 +21,30 @@ public class SimpleCacheEagerConnectionTest
     private readonly ICredentialProvider authProvider = new EnvMomentoTokenProvider("TEST_AUTH_TOKEN");
     
     [Fact]
-    public void SimpleCacheClientConstructor_LazyConnection()
+    public void CacheClientConstructor_LazyConnection()
     {
         var config = Configurations.Laptop.Latest(loggerFactory);
         // just validating that we can construct the client
-        var client = new SimpleCacheClient(config, authProvider, defaultTtl);
+        var client = new CacheClient(config, authProvider, defaultTtl);
     }
     
     [Fact]
-    public void SimpleCacheClientConstructor_EagerConnection_Success()
+    public void CacheClientConstructor_EagerConnection_Success()
     {
         var config = Configurations.Laptop.Latest(loggerFactory);
         config = config.WithTransportStrategy(config.TransportStrategy.WithEagerConnectionTimeout(TimeSpan.FromSeconds(5)));
         // just validating that we can construct the client when the eager connection is successful
-        var client = new SimpleCacheClient(config, authProvider, defaultTtl);
+        var client = new CacheClient(config, authProvider, defaultTtl);
     }
     
     
     [Fact]
-    public void SimpleCacheClientConstructor_EagerConnection_BadEndpoint()
+    public void CacheClientConstructor_EagerConnection_BadEndpoint()
     {
         var config = Configurations.Laptop.Latest(loggerFactory);
         config = config.WithTransportStrategy(config.TransportStrategy.WithEagerConnectionTimeout(TimeSpan.FromSeconds(2)));
         var authProviderWithBadCacheEndpoint = authProvider.WithCacheEndpoint("cache.cell-external-beta-1.prod.a.momentohq.com:65000");
         // validating that the constructor doesn't fail when the eager connection fails
-        var client = new SimpleCacheClient(config, authProviderWithBadCacheEndpoint, defaultTtl);
+        var client = new CacheClient(config, authProviderWithBadCacheEndpoint, defaultTtl);
     }
 }
```

Comment:
Dropped a newline here

Reply:
fixed in https://github.com/momentohq/client-sdk-dotnet/pull/406/commits/4857da7e78172a73f148fa8ccadb2872a3df8765